lib/jnpr/junos/utils/sw.py

```diff
@@ -430,11 +494,20 @@ def _progress(report):
                 _progress(
                     "validating software against current config,"
                     " please be patient ...")
-                v_ok = self.validate(remote_package, dev_timeout=timeout)
-                if v_ok is not True:
-                    return v_ok  # will be the string of output
+                v_ok = self.validate(remote_package, issu, dev_timeout=timeout)
```

Comment:
I think we should not change this. Timeout in general meant for per rpc. So it should be individual for each function call internally too, which is happening as of today.